examples/utils/fee.rs

```diff
@@ -1,5 +1,6 @@
 use bitcoin::{Amount, Network};
 
+#[allow(dead_code)]
```

Comment:
is this necessary?

Reply:
Yes, otherwise `rustc` complains that the fields are unread. Perhaps we should use `#[allow(unused)]`?
Or derive a `Default` implementation for it?

Reply:
ah sorry, I didn't see it was in the examples